en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md

```diff
@@ -68,6 +68,8 @@ By default, the WSO2 API Manager retrieves the client certificate from the **X-W
      certificate_header = "<Header Name>"
      # This property need to be true if MutualSSL connection established between load balancer and gateway.
      enable_client_validation = false
+     #This property needs to be false if the certificate shouldn’t be decoded when it is passed from the load balancer to the gateway.
```

Comment:
can we re-word this. Something like when this is true, Gateway decodes the certificate received in Header.

Reply:
Fixed https://github.com/wso2/docs-apim/pull/2773/commits/96ae8e0ad158afeedcb26772e67d3cc1fa2fedfb